pandas-stubs/core/series.pyi

```diff
@@ -834,7 +859,7 @@ class Series(IndexOpsMixin[S1], NDFrame):
         axis: AxisIndex = ...,
         *args,
         **kwargs,
-    ) -> Series[S1]: ...
+    ) -> Self: ...
```

Comment:
Maybe make this `Series` since we really don't know the type of the `Series` - could be mix of int and float, or one, or the other

Reply:
Thank you, that made it work!